### PR TITLE
[DOCKER] add apt update to test fixture krb5kdc

### PR DESCRIPTION
**Pull request by nknize**
This PR adds an apt update to the krb5kdc test fixture docker file.

### CODE REVIEW

**Comment by nknize** [comment]
start gradle check

**Comment by nknize** [comment]
start gradle check

**Comment by peterzhuamazon** [comment]
Hi @nknize,

Recommend using `apt-get` than `apt` due to:
```
WARNING: apt does not have a stable CLI interface yet. Use with caution in scripts.
```
`apt` is for terminal with beautiful outputs.
`apt-get` is for scripts and give stable, parsable outputs.

File: test/fixtures/krb5kdc-fixture/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,6 @@
 FROM ubuntu:14.04
+RUN apt update -y
+RUN apt upgrade -y
```
Comment (mihirsoni) [inline test/fixtures/krb5kdc-fixture/Dockerfile:3]: np: may be  we can combine in single Docker statement ?  Doesn't matter as it is test but could save some size in docker.

**Review by mihirsoni — APPROVED** [review]
LGTM !!

File: test/fixtures/krb5kdc-fixture/Dockerfile (hunk 1 above)
Comment (nknize) [reply test/fixtures/krb5kdc-fixture/Dockerfile:3]: thx!!  I'll probably also make a follow on change to use `apt-get` instead of `apt`

**Pull request merged**